src/scenarios/mauiandroid/pre.py

```diff
@@ -23,7 +23,7 @@
                 no_restore=False)
 
 # Build the APK
-precommands.execute(['--no-restore', '--source', 'MauiNuGet.config'])
+precommands.execute([])
```

Comment:
Does the build get the `NuGet.config` another way now?

You probably would need at least the dotnet8 feed:
```xml
<?xml version="1.0" encoding="utf-8"?>
<configuration>
  <packageSources>
    <clear />
    <add key="nuget" value="https://api.nuget.org/v3/index.json" />
    <add key="dotnet8" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet8/nuget/v3/index.json" />
  </packageSources>
</configuration>
```

Reply:
It does not seem to, but I will add back in what we had before to cover our bases.